engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java

```diff
@@ -82,9 +82,9 @@ private Engine getEngine() {
         }
     }
 
-    public String renderTemplate(User user, Action action, TemplateInstance templateInstance) {
+    public String renderTemplate(User user, Object data, TemplateInstance templateInstance) {
         return templateInstance
-                .data("action", action)
+                .data("data", data)
```

Comment:
It will likely break, need to fix these - in this step we are going to accept cloud events and transform them right away to the old format for everything else. We will figure out what to do once we start migrating applications.

Reply:
I'm adding both the event and the action variables to the templates - so it's backwards compatible for current events. We will need to migrate each template (once we migrate the application) to start using `event` with the new format.